cdc/cdc/puller/puller.go

```diff
@@ -231,6 +227,7 @@ func (p *pullerImpl) Run(ctx context.Context) error {
 				if err != nil {
 					return errors.Trace(err)
 				}
+				metricTxnCollectCounterResolved.Inc()
```

Comment:
Suggest:
1. Rename to `metricPullOutputCounterResolved`. So as to the underlying prom metric name.
2. Utilize [`kvEventCounter`](https://github.com/tikv/migration/blob/b559233b0f166429b904b8a4390db310a966dacb/cdc/cdc/puller/metrics.go#L21) as the puller input counter. The input & output metrics will help us determine whether there is issue in puller module. For both kv & resolved-ts event.
3. Add the input counter to grafana panel.